### PR TITLE
Fix some minor errors

### PR DESCRIPTION
**Pull request by xujingjing-cmss**
Signed-off-by: xujingjing xujingjing_yewu@cmss.chinamobile.com

**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
some of the errors was not handled correctly.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

### CODE REVIEW

**Comment by daixiang0** [comment]
lgtm

btw, please rename with more meaningful title.

I notice you are working on `keadm debug`, maybe you can put changes together with same subdir or subcommand, which  would be better, go ahead!

**Comment by xujingjing-cmss** [comment]
> lgtm
> 
> btw, please rename with more meaningful title.
> 
> I notice you are working on `keadm debug`, maybe you can put changes together with same subdir or subcommand, which would be better, go ahead!

thx，I'm kind of new to these things. I will try to push more code next time.

File: keadm/cmd/keadm/app/cmd/debug/check.go [hunk 1]
```diff
@@ -285,7 +285,7 @@ func CheckNetWork(IP string, timeout int, cloudhubServer string, edgecoreServer
 	if config != "" {
 		edgeConfig, err := util.ParseEdgecoreConfig(config)
 		if err != nil {
-			err = fmt.Errorf("parse Edgecore config failed")
+			return fmt.Errorf("parse Edgecore config failed")
 		} else {
```
Comment (fisherxu) [inline keadm/cmd/keadm/app/cmd/debug/check.go:289]: The else here can be deleted :)

**Comment by fisherxu** [comment]
> thx，I'm kind of new to these things. I will try to push more code next time.

No problem, all contributions are welcome :) @xujingjing-cmss

**Comment by fisherxu** [comment]
And please squash the commits.

**Comment by xujingjing-cmss** [comment]
> And please squash the commits.

It seem that one of the origin commit has been recommited by myself and I don't know how to reset that.
chose to close this.

File: keadm/cmd/keadm/app/cmd/debug/check.go (hunk 1 above)
Comment (xujingjing-cmss) [reply keadm/cmd/keadm/app/cmd/debug/check.go:289]: I think the else can't be deleted.The variable cloudhubServer need to be set a default value if it's not assigned.

**Pull request closed without merging**